.env.dist

```diff
@@ -0,0 +1,12 @@
+# Database Configuration
```

Comment:
Two things:

- Can this be moved to after the bot config
- Can a space be added between the `#` and the env var name for all commented out env vars